source/_components/switch.arest.markdown

```diff
@@ -46,6 +46,7 @@ Configuration variables:
 - **name** (*Optional*): Let you overwrite the name of the device. By default *name* from the device is used.
 - **pins** array (*Optional*): An array with all used pins.
   - **name** (*Required*): The name of the pin to use in the frontend.
+  - **invert** (*Optional*): If the logic for on/off should be inverted. 1 = off and 0 = on
```

Comment:
Please add the default value. The type (bool) would be nice, too. A `{% configuration %}` block would be awesome. :-)

Reply:
No no, I was trying to call out a reference to the aREST code. (Arduino) 1 or HIGH is off and 0 or LOW is on.

Reply:
Aah check

@w1ll1am23, are you planning on changing/updating this PR, based on the feedback by @syssi ?

Reply:
@frenck yes, didn't get a chance yesterday. I'll try to get it done later today.